### ISSUE TITLE
Segmentation and self-supervised training data 

### ISSUE THREAD

**Issue by letchke** [jingpeilu/CtRNet-robot-pose-estimation#6]
Hi  @jingpeilu and @tristinxie,

really cool paper! I have two data related questions which I couldn't answer myself while studying your paper:

1) How do you train the segmentation model? You write it's pretrained on synth. data on the respective robot. However, the links and data based on your  `README.md` do not consist of segmentation masks. Did you generate your own masks/ data?

2) On which data do you train when you switch to the self-supervised stage of the model training? Do you train on a subset of the test data or did you generate your own "real-world" datasets for this stage?

Would appreciate hearing back from you!

**Comment by jingpeilu** [comment]
Thanks for liking our work!

Regarding your question, here are some details:
1. To create the ground-truth robot masks, we used the same renderer that we used for self-supervised training, along with the ground-truth pose data. You can find more information in Section 6.1 of our paper, which is available at https://arxiv.org/pdf/2302.14332.pdf.
2. During the self-supervised stage, we generated a dataset to train the baxter model. We trained the panda model on the test data since we didn't have a panda robot at the time. Ideally, it would be better to generate a real dataset of your robot for self-supervised training.

**Comment by letchke** [comment]
Thank you so much!

1. Yes, I found this (and implemented as well already) after noticing that I've got the paper version w/ o the appendix!
2. Okay, that's what I have assumed. Coming from the supervised learning domain, it doesn't feel natural to train on a test set, that's why I wanted to clarify this^^
3. Just one more question: Has there been a similar ablation study about the number of pretraining images on the Panda synth data set? Otherwise, I assume you've pretrained for 1000 epochs on all the ~100000 pictures from the Panda synthetic train data set with domain randomization?

**Comment by jingpeilu** [comment]
Yes. We used all Panda synthetic data for pretraining.

**Comment by KJZhuAutomatic** [comment]
Hi, nice work. Can you share me the weight which is pretrained in the Panda synthetic data?
In the weight.zip, just found the weights after self-supervised. 
My email, 984443651@qq.com Thanks.

**Comment by letchke** [comment]
@jingpeilu I would also still be interested in the pretrained weights on the Panda, if available?
Additionally, I'd like to know if you self-supervised on the whole real-world Panda test dataset or just a subset of it? In case of a subset; how big was it? Thanks a lot in advance :)
